### PR TITLE
Towards ertp (works now)

### PR DESCRIPTION
**Pull request by erights**
This pull request is a non-working state and so will not be submitted. But I'm about to be unavailable for several hours and would like you to have a look. Thanks.

### CODE REVIEW

**Comment by dckc** [comment]
ertp = ???

I read thru the diffs and I don't see it spelled out.

**Comment by erights** [comment]
@dckc 

ERTP === Electronic Rights Transfer Protocol

http://www.erights.org/javadoc/net/ertp/package-summary.html
http://www.hpl.hp.com/techreports/2006/HPL-2006-5.html
http://www.erights.org/smart-contracts/ertp/money.html

More soon...

File: examples/contract/contractTest.js [hunk 1]
```diff
@@ -39,18 +39,18 @@ async function mintTest() {
...
-  const contractSrc = `${trivContract}`;
+  const trivContractMaker = (seven, one) => (whiteP, blackP) => seven + one;
+  const contractMakerSrc = `${trivContractMaker}`;
 
-  const tokensP = Vow.resolve(contractHostP).e.setup(contractSrc);
+  const tokensP = Vow.resolve(contractHostP).e.setup(contractMakerSrc, 2);
 
   const whiteTokenP = tokensP.then(tokens => tokens[0]);
-  contractHostP.e.play(whiteTokenP, contractSrc, 0, {});
+  contractHostP.e.play(whiteTokenP,
```
Comment (erights) [inline examples/contract/contractTest.js]: Also need negative tests for the common maker arguments

File: examples/contract/simple_auction.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/*global define def*/
```
Comment (erights) [inline examples/contract/simple_auction.js]: This file is way stale, untested, and unlikely to work. It should be removed or moved where it won't be confused with working files.

**Comment by erights** [comment]
This PR is way out of date and superseded by https://github.com/Agoric/ERTP . Closing.

**Pull request closed without merging**